fecgov/fec-cms
# [DO NOT MERGE] Feature/3509 presidential map 2

## Summary

- Resolves #3509 

The bubbles/circles are in and working, though not styled well. They're clickable but don't really do anything. Also, rolling over them or the state fills will change their layers; I'll deal with that during #3517 

## Impacted areas of the application
This map **AND WCCF - DO NOT MERGE**

## Screenshots
The circles in place (with a black stroke applied)
![image](https://user-images.githubusercontent.com/26720877/75186002-149d3c00-5715-11ea-9137-98f45bb0569c.png)

## Related PRs


## How to test
Pull, build, and run as usual. The circles should appear when the map style is changed from gradient to bubbles. The circles should change as the election year is changed, too—though most values will remain in the same ranges.
____
